### PR TITLE
Minor tweaks to README

### PR DESCRIPTION
**Pull request by arku**
## :construction_worker: Purpose

To improve README

## :rocket: Changes

Fixes minor typos and grammar issues in README

## :warning: Breaking Changes

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,15 +76,15 @@ Please note that support for the I2P Interface is very experimental at the momen
...
 Temporal provides a first of its kind API that outweighs every other third-party IPFS and decentralized storage API on the market so you can fully leverage all the benefits the distributed web has to offer.
 
 ## Funding
 
 Currently the project is paid for by RTrade Technologies Ltd, and we will *not* be doing an ICO. Funding is derived from private investment, mining farm profits, as well as purchasing of RTC.
 
-Should you wish to consider donations, or private investment email admin@rtradetechnologies.com.
+Should you wish to consider donations, or private investment, email admin@rtradetechnologies.com.
 
 Should you wish to contribute not just to TEMPORAL, but to the overall success of RTrade and our platform, you may purchase RTC for ETH from our [RTCETH Smart Contract](https://etherscan.io/address/0x40e68e3F58b9C1928954BEe5dEcC09A45aA531f8#code)
```
Comment (postables) [inline README.md]:
```suggestion
Should you wish to contribute not just to Temporal, but to the overall success of RTrade and our platform, you may purchase RTC for ETH from our [RTCETH Smart Contract](https://etherscan.io/address/0x40e68e3F58b9C1928954BEe5dEcC09A45aA531f8#code)
```

File: README.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ Selected Content:
...
+One of the big concerns with IPFS, and even cloud data storage in general is encryption. As IPFS doesn't yet support native data encryption, we allow users to encrypt their data using AES256-CFB. While this is better than storing data without encryption on IPFS, there are still some concerns with encrypted data storage on IPFS. Namely, if anyone is ever able to discover the content hash and pin the content, it will always be available to them. This is of big concern when using encryption algorithms as it is theoretically possible for someone to persist that data within their own storage system until the desired ciphers are broken, and they can crack the encryption algorithm. If this is something that is of concern to you, an even better solution is to encrypt your data, and store in on private networks. We have plans to eventually migrate to AES256-GCM which is more secure than AES256-CFB, and allow encryption of data with IPFS keys.
```
Comment (postables) [inline README.md]: Not sure if this is just git diff being weird, but it doesn't look like anything was changed for this section?

Comment (arku) [reply README.md]: @postables and even better solution -> _an_ even better solution

Comment (postables) [reply README.md]: :+1:

**Pull request merged**